Samples/MonitorProgress/CudaProgress.cs

```diff
@@ -13,6 +13,7 @@
 using ILGPU.Runtime;
 using ILGPU.Runtime.Cuda;
 using System;
+using System.Diagnostics.CodeAnalysis;
 
 #pragma warning disable CA2213
```

Comment:
I think then we can remove the global oppression, right?

Reply:
Yes, it should be possible to remove the warning suppression.